eslint.config.js

```diff
@@ -407,6 +413,7 @@ export default defineConfig([
           ],
         },
       ],
+      '@typescript-eslint/no-floating-promises': 'error',
```

Comment:
This rule's super helpful in tests. More than a few times I've found myself banging my head against the wall because I forgot to await `expect()` call.